rtgui/edit.h

```diff
@@ -152,6 +145,59 @@ class EditBuffer;
  *
  */
 
+
+
+class ObjectMOBuffer
+{
+private:
+
+    // Used to draw the objects where the color correspond to the object's ID, in order to find the correct object when hovering
```

Comment:
My intention was initially to handle lots of complex shape like curves with different thickness on screen. As you might have understood or seen, the MouseOver objects are usually thicker than their visual representation. Of course, we can still handle the thickness difference for basic shapes, but I don't know how to deal with curves or with complex shapes like polylines.

I don't think that the memory consumption of this mechanism should be a concern, I'm more concerned by the performance and the versatility of this tool. Updating the ObjectMOBuffer takes time, then we only read the value under the cursor. But in your opinion, should we switch to the `contains` method ?

About using `FORMAT_RGB16_565`, yes, that should be better. Like the dumb guy I am, I was looking for an FORMAT_A16, and found a silly trick to circumvent its absence :)